frontend-instructor/src/components/courseManagement/VideoForm.jsx

```
@@ -34,7 +34,7 @@ const VideoForm = ({
       const videoParams = {
         title: values.title,
         url: values.url,
-        length: parseInt(values.length, 10)
+        length: parseInt(values.length) * 60
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)